app/models/setting/remote_execution.rb

```diff
@@ -67,7 +67,12 @@ def self.load_defaults
         self.set('remote_execution_cleanup_working_dirs',
                  N_('When enabled, working directories will be removed after task completion. You may override this per host by setting a parameter called remote_execution_cleanup_working_dirs.'),
                  true,
-                 N_('Cleanup working directories'))
+                 N_('Cleanup working directories')),
+        self.set('remote_execution_cockpit_href',
+                 N_('Where to find the Cockpit instance for the Web Console button.  By default, no button is shown.'),
```

Comment:
would be good to put an example of expected format, so that the user knows that he needs to put '%{host}' in there

Reply:
Do want to keep using %{host}, or should we rather switch to the templating engine that Foreman uses in other places?

Reply:
I think it would be unnecessary complex,'%{host}' is good enough IMO